src/helper.js

```diff
@@ -67,34 +67,37 @@ export const changeInputPlaceHolder = (event) => {
   if (event === 'Enrollment Year') {
     placeHolder = 'E.g 2021';
   }
-  
-return placeHolder;
+
+  return placeHolder;
 };
 
 const validateName = (inputString) => {
-  const reg = /^[\p{L}]+$/u;
-  const isValid = reg.test(inputString);
+  const reg = /^[\p{L} ]+$/u;
+  const isValid = inputString.length === 0 ? true : reg.test(inputString);
```

Comment:
This line can be simply put as `const isValid = inputString.length === 0 || reg.test(inputString);` looks abit more readable. 
I have also noticed that the validation functions are all similar, with only `inputString` and `reg`, I suggest that this becomes one function that you simply call with different inputs. That would reduce on the amount of code considerably